lib/redis/commands/sets.rb

```diff
@@ -154,6 +154,19 @@ def sinterstore(destination, *keys)
         send_command([:sinterstore, destination].concat(keys))
       end
 
+      # This command is similar to SINTER, but instead of returning the result set,
+      #   it returns just the cardinality of the result.
+      #
+      # @param [String, Array<String>] keys keys pointing to sets to intersect
+      # @param [Interger] limits number of matches searched for
+      # @return [Array<String>] cardinality of the intersection
```

Comment:
Isn't it return Integer type?
Seems to need changing it
```suggestion
      # @return [Integer] cardinality of the intersection
```